src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js

```diff
@@ -11,16 +13,33 @@ export class DocumentEItems extends Component {
     element.scrollIntoView({ behavior: 'smooth' });
   };
 
+  onClickEItemRequestLink = () => {
+    const medium = 'ELECTRONIC';
+    return {
+      pathname: FrontSiteRoutes.documentRequestForm,
+      state: { medium },
+    };
+  };
+
   render() {
     const { eitems } = this.props;
 
-    return eitems.total > 0 ? (
+    return (
       <>
         <Header as="h3">Access online</Header>
-        <DocumentLinks eitems={eitems} />
+        {eitems.total > 0 ? (
+          <DocumentLinks eitems={eitems} />
+        ) : (
+          <>
+            No resources available. <br />
+            <Link to={this.onClickEItemRequestLink()}>
+              Request an electronic item.
+            </Link>
+          </>
```

Comment:
```suggestion
            No electronic resources currently available. 
            <Link to={this.onClickEItemRequestLink()}>
              Request a new electronic copy.
            </Link>
          </>
```

Reply:
Was this taken into account?

Reply:
I would add a question mark at the end of the sentence in the link to indicate the intent. What do you think?

Reply:
I don't think it's necessary since we are using the same format (without question marks) in the footer, the help menu and the search page.